src/H5win32defs.h

```diff
@@ -169,11 +168,11 @@ extern "C" {
     H5_DLL char* Wgetlogin(void);
     H5_DLL int c99_snprintf(char* str, size_t size, const char* format, ...);
     H5_DLL int c99_vsnprintf(char* str, size_t size, const char* format, va_list ap);
-    H5_DLL int Wnanosleep(const struct timespec *req, struct timespec *rem);
     H5_DLL herr_t H5_expand_windows_env_vars(char **env_var);
     H5_DLL const wchar_t *H5_get_utf16_str(const char *s);
     H5_DLL int Wopen_utf8(const char *path, int oflag, ...);
     H5_DLL int Wremove_utf8(const char *path);
+    H5_DLL int H5_get_win32_times(H5_timevals_t* tvs);
```

Comment:
This is probably missing in develop, too. I'll have a PR up for a few things I noticed while working on the Windows code.